src/ssmDocument/wizards/publishDocumentWizard.ts

```diff
@@ -0,0 +1,347 @@
+    label: string
+    description?: string
+    detail?: string
+    documentType: SSM.DocumentType
+}
+
+export interface UpdateDocumentQuickPickItem {
+    label: string
+    description?: string
+    detail?: string
+    alwaysShow: boolean
+    name?: string
+}
+
+export class DefaultPublishSSMDocumentWizardContext extends WizardContext implements PublishSSMDocumentWizardContext {
+    private readonly helpButton = createHelpButton(localize('AWS.command.help', 'View Documentation'))
+    private documents: SSM.Types.DocumentIdentifierList | undefined
+    private readonly ssmDocumentClient: SsmDocumentClient
+
+    public constructor(private readonly defaultRegion: string) {
+        super()
+        this.ssmDocumentClient = ext.toolkitClientBuilder.createSsmClient(this.defaultRegion)
+    }
+
+    public async loadSSMDocument(): Promise<void> {
+        if (!this.documents) {
+            this.documents = await toArrayAsync(
+                this.ssmDocumentClient.listDocuments({
+                    Filters: [
+                        {
+                            Key: 'Owner',
+                            Values: ['Self'],
+                        },
+                    ],
+                })
+            )
+        }
+    }
+
+    // TODO: Uncomment code when supporting more document types in future
+    public async promptUserForDocumentType(): Promise<SSM.DocumentType | undefined> {
+        // const documentTypeItems: DocumentTypeQuickPickItem[] = [
+        //     {
+        //         label: localize('AWS.ssmDocument.publishWizard.documentType.automation.label', 'Automation'),
+        //         documentType: 'Automation',
+        //     },
+        // ]
+
+        // const quickPick = picker.createQuickPick<DocumentTypeQuickPickItem>({
+        //     options: {
+        //         ignoreFocusOut: true,
+        //         title: localize('AWS.ssmDocument.publishWizard.documentType.title', 'Select document type'),
+        //     },
+        //     buttons: [vscode.QuickInputButtons.Back],
+        //     items: documentTypeItems,
+        // })
+
+        // const choices = await picker.promptUser({
+        //     picker: quickPick,
+        //     onDidTriggerButton: (button, resolve, _reject) => {
+        //         if (button === vscode.QuickInputButtons.Back) {
+        //             resolve(undefined)
+        //         }
+        //     },
+        // })
+        // const picked = picker.verifySinglePickerOutput(choices)
+
+        // return picked ? picked.documentType : undefined
+        return 'Automation'
+    }
+
+    public async promptUserForDocumentName(): Promise<string | undefined> {
+        const inputBox = input.createInputBox({
+            options: {
+                title: localize('AWS.ssmDocument.publishWizard.ssmDocumentName.title', 'Name your document'),
+                ignoreFocusOut: true,
+            },
+            buttons: [this.helpButton, vscode.QuickInputButtons.Back],
```

Comment:
help button doesn't appear to be handled

Reply:
Removing help button